web/src/components/TestSpecDetail/Header.tsx

```diff
@@ -69,13 +71,17 @@ const Header = ({
 
       <Divider />
 
-      <Editor
-        type={SupportedEditors.Selector}
-        editable={false}
-        basicSetup={{lineNumbers: false}}
-        placeholder="Selecting All Spans"
-        value={title || 'All Spans'}
-      />
+      {title && <S.Title>{title}</S.Title>}
```

Comment:
In the video, there seems to be a weird misalignment between these two. Could we also increase either the font-weight or size so the title looks more prominent?